packages/ui/src/Checkbox.tsx

```diff
@@ -26,7 +25,7 @@ export type CheckboxExtraProps = {
   classNameLabel?: string
 }
 
-type CheckboxProps = CheckboxCoreProps & CheckboxExtraProps & DataTestProp
```

Comment:
I would suggest leaving `DataTestProp`, and pass its value to the top-level `div`. We use it in MC to find the elements in E2E tests.

Reply:
@aigoncharov, good. Should we then do this for all of the components?

Reply:
I would say so, yeah.